docs/design/data/armor/body.yaml

```
@@ -0,0 +1,120 @@
+source: docs/_incoming/XM Design Bible - Armor.tsv
+slot: Body
+tsv: |
+  Type	Level	Item Name	Source	DEF	HP	EP	ACC	ATK	EVA	MGT	PER	VIT	WIL	AGI	SOC	BST	BRW	ELM	HNT	KPR	MDR	NGT	TEC
+  Body	1	Aurion Alloy Harness	Armorcraft	4										x	x			x		x
+  Body	1	Aurion Alloy Harness +1	Armorcraft	5										x	x			x		x
+  Body	1	Etherwoven Robe	Armorcraft	3											x	x	x		x	x	
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)